lib/environments/meteorEnvironment.js

```diff
@@ -1,22 +1,46 @@
+import fs from 'fs';
+import path from 'path';
+
+import findPackageDependencies from '../findPackageDependencies';
+
+const coreModules = [
+  'meteor/accounts-base',
+  'meteor/blaze',
+  'meteor/check',
+  'meteor/ddp-client',
+  'meteor/ddp-rate-limiter',
+  'meteor/ejson',
+  'meteor/email',
+  'meteor/http',
+  'meteor/check',
+  'meteor/meteor',
+  'meteor/mongo',
+  'meteor/random',
+  'meteor/reactive-var',
+  'meteor/session',
+  'meteor/templating',
+  'meteor/tracker',
+];
+
+function meteorPackageDependencies(
+  { config }: Object
+): Array<string> {
+  const meteorPackagesPath = path.join(config.workingDirectory, '.meteor/packages');
```

Comment:
I'm trying to find documentation about what the syntax of this file looks like. Based on the regexp it seems to be a simple text file with a package per line. It could help if there was a comment here explaining things, or something in the commit message.